h/accounts/views.py

```diff
@@ -680,9 +680,9 @@ def dismiss_sidebar_tutorial(request):
 def includeme(config):
     config.add_route('login', '/login')
     config.add_route('logout', '/logout')
-    config.add_route('register', '/register')
+    config.add_route('signup', '/signup')
     config.add_route('activate', '/activate/{id}/{code}')
-    config.add_route('forgot_password', '/forgot_password')
+    config.add_route('forgot-password', '/forgot-password')
```

Comment:
The failing tests will also be fixed when you change the route name back to `forgot_password`.